nessie/golden/lakectl_doctor_short_secret_access_key.golden

```diff
@@ -0,0 +1,6 @@
+secret_access_key value looks suspicious...
+It looks like you have a problem with your '.lakectl.yaml' file. 
+It is possible that the access_key_id' or 'secret_access_key' you supplied are wrong 
+
+
+Error executing command.
```

Comment:
Same as [this comment](https://github.com/treeverse/lakeFS/pull/2948/files#r806420841)